spring-cloud-stream-schema/src/main/java/org/springframework/cloud/stream/schema/client/config/SchemaRegistryClientProperties.java

```diff
@@ -22,7 +22,7 @@
  * @author Marius Bogoevici
  * @author Vinicius Carvalho
  */
-@ConfigurationProperties(prefix = "spring.cloud.stream.schemaRegistryClient")
+@ConfigurationProperties(prefix = "spring.cloud.stream.schema-registry-client")
```

Comment:
Actually, this is my mistake see https://github.com/spring-cloud/spring-cloud-stream/pull/959#discussion_r116241186 as well